h2o-r/tests/runitUtils/utilsR.R

```diff
@@ -113,10 +113,13 @@ function(cur.dir, root, root.parent = NULL) {
 default.packages <-
 function() {
   to_require <- c("jsonlite", "RCurl", "RUnit", "R.utils", "testthat", "ade4", "glmnet", "gbm", "ROCR", "e1071",
-                  "tools", "statmod", "fpc", "cluster")
+                  "tools", "statmod", "fpc", "cluster", "survival")
   if (Sys.info()['sysname'] == "Windows") {
     options(RCurlOptions = list(cainfo = system.file("CurlSSL", "cacert.pem", package = "RCurl"))) }
-  invisible(lapply(to_require,function(x){require(x,character.only=TRUE,quietly=TRUE,warn.conflicts=FALSE)}))
+  invisible(lapply(to_require,function(x) {
+    if (!require(x,character.only=TRUE,quietly=TRUE,warn.conflicts=FALSE))
+      warning(paste("Package", x, "is not available."), call. = FALSE)
```

Comment:
Mainly to make it easier for us to find out why some test might not work. We import packages and if unsuccessful we used to silently fail, after this change we should see that some package is not available (and R names are sometimes quite obscure so it's hard to guess if we are missing a package or badly exporting something or both (`survfit.H2OCoxPHModel`)).